### PR TITLE
DataFusion 16.0.0 documentation

### PR DESCRIPTION
**Pull request by andygrove**

### CODE REVIEW

**Review by alamb — APPROVED** [review]
Thanks @andygrove  

This reminds me we could/should probably work on updating the documentation more. I have it on my wishlist and will work on it when I have time.

File: datafusion/user-guide/configs.html [hunk 1]
```diff
@@ -321,116 +321,130 @@ <h1>Configuration Settings<a class="headerlink" href="#configuration-settings" t
...
-<th class="head"><p>type</p></th>
 <th class="head"><p>default</p></th>
 <th class="head"><p>description</p></th>
 </tr>
 </thead>
 <tbody>
+<tr class="row-even"><td><p>datafusion.catalog.create_default_catalog_and_schema</p></td>
+<td><p>true</p></td>
+<td><p>Number of partitions for query execution. Increasing partitions can increase concurrency. Defaults to the number of cpu cores on the system.</p></td>
+</tr>
```
Comment (DDtKey) [inline datafusion/user-guide/configs.html:332]: Is that correct statement? Looks wrong and default value is bool 🤔

Comment (DDtKey) [reply datafusion/user-guide/configs.html:332]: см: @andygrove @alamb

Comment (alamb) [reply datafusion/user-guide/configs.html:332]: It sure looks incorrect to me. I think it is an issue upstream: https://github.com/apache/arrow-datafusion/blob/5d4038a8463a575328bedbc22b32456f5dcd562c/datafusion/common/src/config.rs#L157-L158


Nice catch @DDtKey  -- I'll make a PR

Comment (alamb) [reply datafusion/user-guide/configs.html:332]: https://github.com/apache/arrow-datafusion/pull/5032

**Pull request merged**